apps/apputil.hpp

```diff
@@ -177,6 +196,13 @@ struct OutString
 };
 
 
+struct OutBool
+{
+    typedef bool type;
+    static type process(const options_t::mapped_type& i) { return CheckTrue(i); }
+};
+
+
```

Comment:
@ethouris Added `OutBool` option type. To review.

Reply:
This looks ok.